### ISSUE TITLE
Submitted Python Solution on Exercism (05/19)

### ISSUE THREAD

**Issue by anitabe404** [studydash/cards#49]
## Motivation/Problem Statement 💭❓:
Python, python, and more python. And some Data Analysis.

---

## Today's Goal/Solution 🥅:
Learning about Dictionaries.

---

## Result 🍓🍌🍉:
Today, I watched one of the lecture videos from the Python Data Structures course. It introduced dictionaries in Python. I then decided to do a problem on exercism.io. I got all the tests to pass and submitted my solution for mentor feedback. I'm a bit nervous about it because I took a more complex approach to it than I could have, and I'm hoping it wasn't too much. I'm still getting a feel for simplicity vs robust solutions. 

---

## Observations & Next Steps 🔭👣:
I feel like I'm making really good progress with Python, both in terms of the course and my general understanding of it. I find that I like it better than when I first started. I also enjoy doing Python more than I did the stuff from The Odin Project. I'm not fully sure why though. Maybe it's because the project is smaller and I understand Python better than HTML/CSS. I think backend engineering may be in my DNA. We shall see when I finally sit down to learn SQL. 😂

---

## References 🔗:
None

---

## Extra/Fun (Optional) 🎈🎉🥳:
I went for a walk and the weather was gorgeous! We actually have Spring in FL this year instead of the hot, humid springs we've been having. I welcome it!

**Comment by mccurcio** [comment]
Hey Anita,
Did you really find a "Solution on Exercism"? ;)
Do you think Python or your computer is **possessed**? lol

**Comment by anitabe404** [comment]
@mccurcio Yes, my computer is definitely possessed. Mostly because it's the first Mac I've owned and I still barely know how to use it after 3 years. 🤪🤪

Do you use exercism? It's a really cool site. You might like it. [exercism.io](https://exercism.io).

**Comment by mccurcio** [comment]
OOOOhhh,
I thought you misspelled EXERCISE and were thinking EXORCISM!
LOL

Never owned a MAC. I get some confused by them when I start "RIGHT"
clicking, ARRRGGGH.

Stand Back! Linux only for the past ten years. haha
